src/app/globals.css

```
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Anton&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans&display=swap');
```

You can reduce this to a single request
```suggestion
@import url('https://fonts.googleapis.com/css2?family=Anton&family=DM+Sans&display=swap');
```